File: RSPAClient/Core/AtPar.POCOEntities/MT_ATPAR_LOC_GROUPS.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_ATPAR_LOC_GROUPS
    {
        public string ORG_GROUP_ID { get; set; }
        public string LOC_GROUP_ID { get; set; }
        public string LOC_DESCR { get; set; }
        public bool STATUS { get; set; }
        public System.DateTime LAST_UPDATE_DATE { get; set; }
        public string LAST_UPDATE_USER { get; set; }
        public string LAST_CLIENT_ADDRESS { get; set; }

        //Ignore
        public int CHK_VALUE { get; set; }
        public string USER_ID { get; set; }
        public string CHK_ALLOCATED { get; set; }
        public int ROWINDEX { get; set; }
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/MT_ATPAR_LOC_GROUPS.ts
export class MT_ATPAR_LOC_GROUPS {
    public ORG_GROUP_ID: string;
    public LOC_GROUP_ID: string;
    public LOC_DESCR: string;
    public STATUS: boolean;
    public LAST_UPDATE_DATE:  Date;
    public LAST_UPDATE_USER: string;
    public LAST_CLIENT_ADDRESS: string;

    //Addition field for front-end handling
    public checkvalue: boolean;
    public DEFAULT_PRINTER: string;
    public COUNT_FLAG: string;
    public ALLOW_SIC_CONSIGN: string;
    public CHK_VALUE: number;
    public CHK_ALLOCATED: number;
    public USERNAME: string;
    public ROWINDEX: number;

    public CHK_ALLOCATED_CNTFLG: string;
    public ROWINDEXFORCNTFLG: string;
    public CHK_ALLOCATED_CONSFLG: string;
    public ROWINDEXFORCONSFLG: string;
}